cisco_umbrella_dns/manifest.json

```diff
@@ -0,0 +1,53 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "9f98de10-9c98-4601-ae36-cbe25c4be018",
+    "app_id": "cisco-umbrella-dns",
+    "display_on_public_website": true,
```

Comment:
Let's set this to `false` for now so we can deploy the integration for internal testing without making it publicly visible.

Reply:
Done